calibrate.py

```diff
@@ -540,8 +540,9 @@ def calibrate(self):
                 left = dai.CameraBoardSocket.RIGHT
                 right = dai.CameraBoardSocket.LEFT
 
-            calibration_handler = dai.CalibrationHandler()
-            calibration_handler.setBoardInfo(self.board_config['board_config']['name'], self.board_config['board_config']['revision'])
+            calibration_handler = self.device.readCalibration2()
```

Comment:
readCalibration2 throws if EEPROM is empty. Make sure to catch / notify / handle that case.
Also, make sure to flash both USER & FACTORY sections and compare those to be equal